### ISSUE TITLE
default time does not set seconds

### ISSUE THREAD

**Issue by timbonz** [xdan/datetimepicker#366]
default time does not set seconds - the seconds is the current clock seconds value
jquery.datetimepicker.js needs xxx.setSeconds(0); added after all?.some? of the occurrences of xxx.setMinutes(...)

**Comment by resinllc** [comment]
I think I'm having the same issue. I'd like to use:

```{format: 'Y-m-d H:i:s',
   defaultTime: '23:59:59'}
```

This will always output something like: 2015-12-08 23:59:00. It also has erratic behavior. If I set the defaultTime property to something like '05:05:00', it will always just output 00:00:00 as the time.

**Comment by bostonmacosx** [comment]
Agreed...the seconds is not properly supported in this implementation. It is picking up the OSes seconds and not setting it to 00 on click of the time in the picker.

**Comment by chopraaseem** [comment]
Its the same issue which we are also facing it takes OS seconds.

**Comment by dipjoydevpramanik** [comment]
Set out.sec = 0; where ever there is out.sec, Change  _lpad(v.getSeconds(), 2); to  _lpad(0, 2); and v.setSeconds(0). This will set the seconds in datetimepicker to 00.
